src/pg/test/db/dbWrite.test.js

```diff
@@ -0,0 +1,86 @@
+import Graffy from '@graffy/core';
+import sql from 'sql-template-tag';
+import expectSql from '../expectSql.js';
+import pg from '../../index.js';
+import { nowTimestamp } from '../../sql/clauses';
+
+import { PgDb } from '../../db/pool';
+
+jest.mock('../../db/pool');
+const mockClient = {
+  query: jest.fn(),
+  release: jest.fn(),
+};
+
+const mockQuery = jest.fn();
+describe('postgres', () => {
+  let store;
+
+  beforeEach(async () => {
+    jest.useFakeTimers();
+    PgDb.prototype = { write: mockQuery, getClient: mockClient };
```

Comment:
Shouldn't the getClient mock be a function that returns the mockClient object?

But in any case, as you have mocked the .write, it looks like `getClient` never gets called. This may be unnecessary.